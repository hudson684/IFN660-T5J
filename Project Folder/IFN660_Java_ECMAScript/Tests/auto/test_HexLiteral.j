0x1000000
0x5_2
0x7fff_fff
0x8000_0000
0xffff_ffff
0x100000l
0x100000L
0x8000_0000_0000_0000L
0x8000_0000_0000_0000l
0xDada_Cafe
0x00_FF_00_FF