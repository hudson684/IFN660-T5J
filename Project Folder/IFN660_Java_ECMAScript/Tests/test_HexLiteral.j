valid
0x1000000
0x5_2
0x7fff_fff
0x8000_0000
0xffff_ffff
0x100000l
0x100000L
0x8000_0000_0000_0000L
0x8000_0000_0000_0000l
0xDada_Cafe
0x00_FF_00_FF



invalid
523
523l
523L
0_x52
0x_52
0x52_