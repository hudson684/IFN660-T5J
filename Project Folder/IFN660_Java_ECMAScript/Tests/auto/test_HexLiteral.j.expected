IntegerLiteral (16777216)
IntegerLiteral (82)
IntegerLiteral (134217727)
IntegerLiteral (-2147483648)
IntegerLiteral (-1)
IntegerLiteral (1048576)
IntegerLiteral (1048576)
IntegerLiteral (-9223372036854775808)
IntegerLiteral (-9223372036854775808)
IntegerLiteral (-623195394)
IntegerLiteral (16711935)
EOF